packages/rs-drive-abci/src/rpc/core.rs

```diff
@@ -214,7 +235,7 @@ impl CoreRPCLike for DefaultCoreRPC {
     fn get_transactions_are_chain_locked(
         &self,
         tx_ids: Vec<Txid>,
-    ) -> Result<Vec<GetTransactionLockedResult>, Error> {
+    ) -> Result<Vec<Option<GetTransactionLockedResult>>, Error> {
```

Comment:
Should be a Map instead imo, makes things easier to work with.

Reply:
To be honest, I'm not sure we even need this method anymore. @markin-io @codablock let's just remove it since we aren't using it?